CONTRIBUTING.md

```diff
@@ -117,20 +117,18 @@ the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr)
 
 ### 🧐 Linting, formatting, and tests
 
-ZenML is mainly developed using [Poetry](https://python-poetry.org/) as the
-dependency management system. In order to
-install all core dev-dependencies, do:
+To install ZenML from your local checked out files includin all core dev-dependencies, run:
 
 ```
-poetry install
+pip install -e ".[dev]"
```

Comment:
Wouldn't they want the server as well for developing with ZenML?

Reply:
Potentially but not necessarily? I just copied the previous behaviour tbh, which also didn't install the server extra. I'll change it

Reply:
Yeah either add both by default, or explain the different combinations they might want to use depending on what they're doing?